lib/benchee/output/benchmark_printer.ex

```diff
@@ -26,7 +26,16 @@ defmodule Benchee.Output.BenchmarkPrinter do
     suite_information(jobs, config)
   end
 
-  defp system_information(%{erlang: erlang_version, elixir: elixir_version}) do
+  defp system_information(%{erlang: erlang_version,
+                            elixir: elixir_version,
+                            os: os,
+                            num_cores: num_cores,
+                            cpu_speed: cpu_speed,
+                            available_memory: available_memory}) do
+    IO.puts "Operating System: #{os}"
+    IO.puts "CPU Information: #{cpu_speed}"
+    IO.puts "Number of Available Cores: #{num_cores}"
+    IO.puts "Available memory: #{available_memory}"
```

Comment:
Yeah, I think it'll be helpful to see what is output for the calls for the other platforms before deciding where and how to handle the "humanizing" of this number. I'll do some more research today and I'll see if I can try some stuff out. This will be a little tricky to test since I don't have a Linux or Windows machine set up for development, but I guess I can spin up a VM for that.